### PR TITLE
Add instructions configuring VS2015 to compile sln

### PR DESCRIPTION
**Pull request by bbrandt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,8 @@ Message templates are a superset of standard .NET format strings, so any format
   * The destructuring operator (`@`) in front of will serialize the object passed in, rather than convert it using `ToString()`.
   * the stringification operator (`$`) will convert the property value to a string before any other processing takes place, regardless of its type or implemented interfaces.
 * Property names may be suffixed with an optional format, e.g. `:000`, to control how the property is rendered; these format strings behave exactly as their counterparts within the `string.Format()` syntax
+
+### Compiling
+
+Opening serilog-formatting-compact.sln requires VS2015 Update 3 with the .NET Core tooling to be installed.
```
Comment (adamchester) [inline README.md]: Thanks! Could this have the correct sln name please?

**Comment by bbrandt** [comment]
Solution name updated!  Should we include instructions about removing the sdk.version from global.json before opening in VS?  I couldn't find any info anywhere on how to install 1.0.0-preview3-003161, just .NET Core Tooling preview 2 which seemed to be able to compile the project adequately.

BTW, Thanks for contributing the project.  Looks very useful!

**Comment by adamchester** [comment]
Thanks!

I guess we could pull the script out from appveyor.yml and use that for instructions? As you probably know, those tools are still not yet 1.0 stable, so we expect the specific versions to change in the coming weeks

**Comment by bbrandt** [comment]
Thanks for merging!  

I think it's good for now.  When the tooling reaches RTM we can update if necessary.

**Pull request merged**